test/test_helper.rb

```diff
@@ -4,15 +4,15 @@
 
 class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-  fixtures :all     
+  fixtures :all
 
   Time.zone = 'Alaska'
   Time.now
   Date.today
-begin
-  
-rescue Exception
-  
-end
+  begin
+
+  rescue Exception
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?